integration/hurl/tests_ok/http_version_option.hurl

```diff
@@ -3,6 +3,15 @@ GET http://localhost:8000/http_version/10
 http1.0: true
 HTTP 200
 
+
+GET http://localhost:8000/http_version/10
```

Comment:
We can remove this request, in the previous request there is no implicit check (`HTTP 200`) so no need to add an explicit one